regimes/pt/tags.go

```diff
@@ -7,46 +7,12 @@ import (
 	"github.com/invopop/gobl/tax"
 )
 
-// Tax exemption tags
-const (
-	TagExempt        cbc.Key = "exempt"
-	TagOutlay        cbc.Key = "outlay"
-	TagIntrastate    cbc.Key = "intrastate-export"
-	TagImports       cbc.Key = "imports"
-	TagExports       cbc.Key = "exports"
-	TagSuspension    cbc.Key = "suspension-scheme"
-	TagInternalOps   cbc.Key = "internal-operations"
-	TagSmallRetail   cbc.Key = "small-retail-scheme"
-	TagExemptScheme  cbc.Key = "exempt-scheme"
-	TagTobacco       cbc.Key = "tobacco-scheme"
-	TagMargin        cbc.Key = "margin-scheme"
-	TagTravel        cbc.Key = "travel"
-	TagSecondHand    cbc.Key = "second-hand"
-	TagArt           cbc.Key = "art"
-	TagAntiques      cbc.Key = "antiques"
-	TagTransmission  cbc.Key = "goods-transmission"
-	TagOther         cbc.Key = "other"
-	TagFlatRate      cbc.Key = "flat-rate-scheme"
-	TagNonDeductible cbc.Key = "non-deductible"
-	TagConsignment   cbc.Key = "consignment-goods"
-	TagReverseCharge cbc.Key = "reverse-charge"
-	TagWaste         cbc.Key = "waste"
-	TagCivilEng      cbc.Key = "civil-eng"
-	TagGreenhouse    cbc.Key = "greenhouse"
-	TagWoods         cbc.Key = "woods"
-	TagB2B           cbc.Key = "b2b"
-	TagIntraEU       cbc.Key = "intraeu"
-	TagRealEstate    cbc.Key = "real-estate"
-	TagGold          cbc.Key = "gold"
-	TagNonTaxable    cbc.Key = "non-taxable"
-)
-
 // Invoice type tags
 const (
 	TagInvoiceReceipt cbc.Key = "invoice-receipt"
 )
 
-var invoiceTags = []*tax.Tag{
+var invoiceTags = []*tax.KeyDefinition{
```

Comment:
@samlown I noticed that, in IT, you moved the `invoiceTags` to `scenarios.go` and deleted `tags.go`. Do we want to do the same in PT?

Reply:
yes, that does make sense... moving.